### PR TITLE
Added the libzmq3-dev dependency for building in Debian(Ubuntu)

### PR DESCRIPTION
**Pull request by saki-osive**
Hi, I have added the needed pre-requisite for building Grid on Debian(Ubuntu).

### CODE REVIEW

**Comment by KevinAnsleyJohnson** [comment]
> Hi, I have added the needed pre-requisite for building Grid on Debian(Ubuntu).

Have you tried installing `pkg-config` and `protobuf` instead of libzmq? I believe that pkg-config would be a more complete option.

**Comment by saki-osive** [comment]
@KevinAnsleyJohnson I can confirm that the ``libzmq3-dev`` is definitely needed, here's what cargo throws at me when it's not installed and a build is invoked: 
https://gist.github.com/saki-osive/823f86d613c1aebe8ae9fd65e5632d9a

File: README.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ discussion](https://grid.hyperledger.org/community/join_the_discussion.html).
 
 ## Building Grid
 
+Before building, please make sure you have the `libzmq3-dev` package installed in Debian(Ubuntu).
```
Comment (KevinAnsleyJohnson) [inline README.md:49]: Please break this readme line at 72 characters.

**Comment by KevinAnsleyJohnson** [comment]
> @KevinAnsleyJohnson I can confirm that the `libzmq3-dev` is definitely needed, here's what cargo throws at me when it's not installed and a build is invoked:
> https://gist.github.com/saki-osive/823f86d613c1aebe8ae9fd65e5632d9a

I verified that you are correct. Please update the readme (line break at 72 chars) and update your commit to follow the contributing guidelines:
https://grid.hyperledger.org/community/contributing.html

File: README.md (hunk 1 above)
Comment (saki-osive) [reply README.md]: Done, please verify it now.

**Comment by saki-osive** [comment]
@KevinAnsleyJohnson Hi, I have realized that there are several other packages that need to be installed before the project can be built successfully. I have listed them in order in which they are needed as the project starts building. Please verify this too.

**Comment by KevinAnsleyJohnson** [comment]
Note: I'm not a maintainer, but have worked a bit with the maintainers, and know some of what to expect. They will comment on the fact that this should be one commit and not two - so I suggest you undo the two commits, and turn it into one commit. (you will need to force-push your changes to your branch). Also, the commit message should read as a command, so instead of "Added" you should say "Add" - also include a description of your commit in the commit message. I'm available on chat.hyperledger.com if you'd want to discuss.

Thanks!

**Comment by saki-osive** [comment]
@KevinAnsleyJohnson Hi, thank you so much for bringing forth that. Just saw that all the requirements are provided in the ci/grid-dev but there's no pointer that asks the reader to refer to it once, which I believe should be there for a newcomer to the project. I'm closing this and creating a new PR that addresses exactly that.

PR closed as prereqs for the building were present in ci/grid-dev but no reference was provided that it should be read before building.

**Comment by saki-osive** [comment]
@KevinAnsleyJohnson The ci/grid-dev folder is for the containerized version of the build. The fact that exact same dependencies are there as well is because the base image used by the team is ubuntu: focal. However, I believe the dependencies needed for building the project must be clearly mentioned on the README page as the page asks to run `cargo build` from the root directory and doesn't ask to check the ci folder. Even asking to check the ci folder looks vague as it serves a different purpose. Overall, I believe the dependencies needed to build should be clearly mentioned on the README page or there should be a pointer to refer to before invoking the build. 

I will either add a dependencies file or just add them in the README as we discuss further.

**Comment by saki-osive** [comment]
@shannynalayna How to proceed with this?

- A separate file listing all dependencies or,
- All dependencies listed in the README ?

**Pull request closed without merging**